File: C#/begginer/1008.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int num, workedHours;
        double recPerHour;

        num = int.Parse(Console.ReadLine());
        workedHours = int.Parse(Console.ReadLine());
        recPerHour = double.Parse(Console.ReadLine());

        double salary = workedHours * recPerHour;

        Console.WriteLine($"NUMBER = {num}");
        Console.WriteLine($"SALARY = U$ {salary:F2}");

    }

}

File: Java/begginer/1008.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        int num = scanner.nextInt();
        int workedHours = scanner.nextInt();
        double usPerHour = scanner.nextDouble();
        
        System.out.println("NUMBER = " + num);
        System.out.printf("SALARY = U$ %.2f\n", workedHours * usPerHour);
    }
 
}
